webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js

```diff
@@ -26,12 +26,13 @@ const AuditsPage = ({
     onBookmarkClick={search => fetchAndPush({ searchQuery: search, page: 1 })}
     toolbarButtons={
       <Button
+        component="a"
+        className="btn-docs"
         href={getManualURL('4.1.4Auditing')}
-        target="_blank"
         rel="external noreferrer noopener"
-        className="btn-docs"
+        target="_blank"
+        variant="secondary"
       >
-        <Icon type="pf" name="help" />
```

Comment:
Is it better without an icon?

Reply:
@sharvit thanks for the review!
The search bar pf4 applied styles is being discussed in #8743 

There was a request to divide the other buttons adjustments to another PR so only the second commit here is relevant :)